packages/CollapsibleChecklists/src/CollapsibleChecklists.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Heading from "./components/Heading";
+import Group from "./components/Group";
+import Item from "./components/Item";
+
+const propTypes = {
+  children: PropTypes.node.isRequired,
+  onChange: PropTypes.func.isRequired,
+};
+
+const defaultProps = {};
+
+const CollapsibleChecklists = props => {
+  const { children, onChange } = props;
+
+  return (
+    <React.Fragment>
+      {children.map((child, index) => {
+        switch (child.type.displayName) {
+          case Heading.displayName:
+            return <Heading key={`heading${index}`} {...child.props} />; // eslint-disable-line react/no-array-index-key
```

Comment:
The `key` is necessary for React console warning:
```
Warning: Each child in a list should have a unique "key" prop.
```

Reply:
See if that's still there if you just use `child` itself?
It should also disappear for the `Group` case if you use `React.cloneElement` since that should preserve keys.